### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by johancvb**
- Replaced "FemAlert" with ReportAlert
- Removed screenshots in ReadMe
- Added screenshot of homepage on top of ReadMe
- Added code blocks instead of img's
- Added database-text

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,24 +18,56 @@ _by [Sharon Veldman](https://github.com/SharonV33), [Johan van Beusekom](https:/
...
-FemAlert is a platform where people can report any kind of unacceptable behavior in an accessible way, without immediately taking it to the police. FemAlert also lets club or bar owners get in touch with victims to help them and offer support.
+That's where ReportAlert comes in handy. ReportAlert is an initiative, established by Rosanne Verbeek, to protect people from unwanted situations while partying or going out to clubs.
+ReportAlert is a platform where people can report any kind of unacceptable behavior in an accessible way, without immediately taking it to the police. ReportAlert also lets club or bar owners get in touch with victims to help them and offer support.
 
-The platform FemAlert is also made for club owners to give them insights, to see exactly what's going on inside of their clubs. With these insights, club owners can anticipate on the situations to make the nightlife much safer and better in their venues.
+The platform ReportAlert is also made for club owners to give them insights, to see exactly what's going on inside of their clubs. With these insights, club owners can anticipate on the situations to make the nightlife much safer and better in their venues.
```
Comment (veerleprins) [inline README.md]: Not only club owners, but also festival owners 😄

File: README.md [hunk 2]
```diff
@@ -16,24 +18,56 @@ _by [Sharon Veldman](https://github.com/SharonV33), [Johan van Beusekom](https:/
...
 
 Our job is to make a neat-designed and detailed dashboard for both club owners and festival owners to provide them with visualized data about the reports made in/for their clubs.
 
 ## 🔢 Data
 
-FemAlert is still a concept, which means it doesn't have a working application yet. Because it's still a work in progress, we couldn't be provided with real data. Therefore, we had to generate our own dummy data. To do this, we used the online tool mockaroo.com. With Mockaroo you can create mock data, to your own liking. Based on the steps you have to take and the forms you have to fill in while making a new report, we created the dummy data with corresponding data-types and values.
+ReportAlert is still a concept, which means it doesn't have a working application yet. Because it's still a work in progress, we couldn't be provided with real data. Therefore, we had to generate our own dummy data. To do this, we used the online tool mockaroo.com. With Mockaroo you can create mock data, to your own liking. Based on the steps you have to take and the forms you have to fill in while making a new report, we created the dummy data with corresponding data-types and values.
```
Comment (veerleprins) [inline README.md:30]: Nice 👍🏼

File: README.md [hunk 3]
```diff
@@ -16,24 +18,56 @@ _by [Sharon Veldman](https://github.com/SharonV33), [Johan van Beusekom](https:/
...
+    "email": "lopyb@amazon.de",
+    "need_help": false,
+    "make_contact": true,
+    "club_id": 18
+  }
+```
+
+Club data:
+
+```
```
Comment (veerleprins) [inline README.md:60]: If you write javascript in the ``` and before the {, then markdown should know that it is javascript code 😄

**Review by veerleprins — CHANGES_REQUESTED** [review]
Looks good! Would you like to look only at the small dots of feedback yet? 😸

**Review by SharonV33 — APPROVED** [review]
great work!

**Review by veerleprins — APPROVED** [review]
👍🏼

**Pull request merged**